connectors/src/connectors/notion/index.ts

```diff
@@ -44,10 +45,11 @@ export async function createNotionConnector(
     throw new Error("NANGO_NOTION_CONNECTOR_ID not set");
   }
 
-  const notionAccessToken = (await nango_client().getToken(
-    NANGO_NOTION_CONNECTOR_ID,
-    nangoConnectionId
-  )) as string;
+  const notionAccessToken = await getAccessTokenFromNango({
```

Comment:
why not extend Nango() and override getToken() ?

Reply:
Why would I do this ?

Reply:
I honestly see 0 reason for doing this ? Nango isn't our lib